demo/greeting/src/main/scala/greeting/runtime/implicits.scala

```diff
@@ -40,5 +43,31 @@ object implicits {
 
   implicit val grpcServerHandler =
     new GrpcServerHandler[Future] andThen new GrpcConfigInterpreter[Future]
+}
+
+trait ClientImplicits {
+
+  import freestyle.rpc.client._
+  import freestyle.rpc.client.implicits._
+  import freestyle.rpc.client.handlers.{ChannelMHandler, ClientCallsMHandler}
+
+  implicit val ec: ExecutionContext = ExecutionContext.Implicits.global
+
+  val channelFor: ManagedChannelFor = ManagedChannelForAddress(host, portNode1)
+
+  val channelConfigList: List[ManagedChannelConfig] = List(UsePlaintext(true))
+
+  implicit def channelMHandler[F[_]]: ChannelM.Op ~> Future =
+    new ChannelMHandler[Future] andThen
+      new ManagedChannelInterpreter[Future](channelFor, channelConfigList)
+
+  implicit val clientCallsMHandler: ClientCallsM.Op ~> Future = new ClientCallsMHandler[Future]
+
+}
+
+object implicits {
```

Comment:
I think users may be confused with this. Every time you import something with the name `implicits` users may go for `implicits._` right away. Since this is a demo is not critical but I think we should try avoiding nesting further implicit scopes once we hit the `implicits` one. For this case I would just have:
`object implicits extends ServerImplicits with ClientImplicits`